AutoscoperM/AutoscoperM.py

```diff
@@ -444,6 +444,7 @@ def onGeneratePartialVolumes(self):
             mainOutputDir = self.ui.mainOutputSelector.currentPath
             tiffSubDir = self.ui.tiffSubDir.text
             tfmSubDir = self.ui.tfmSubDir.text
+            trackingSubDir = self.ui.trackingSubDir.text
```

Comment:
For reference, the corresponding directory is defined here in Autoscoper. See https://github.com/BrownBiomechanics/Autoscoper/blob/dd32d16390c71b46641173aac9ad311ead9dfd39/autoscoper/src/ui/ui-files/AdvancedOptionsDialog.ui#L196-L204

Reply:
If the tracking directory is not used in the `onGeneratePartialVolumes` function, I suggest to remove references to it.

My understanding is that Autoscoper should already take care of creating the `Tracking` directory and any value we would choose in the AutoscoperM UI would not have any impact on where Autoscoper is saving those `.tra` files.

Once we work on adding export to `.tra`, we should then take care of making the corresponding directory exists.

Reply:
Sounds good. As discussed, I'll remove the references to the path, but keep the Tracking Subdirectory field in the UI as disabled.